src/utils/convertToSignableToken.ts

```diff
@@ -0,0 +1,35 @@
+import { SignableToken } from '../api';
+import { TokenAmount } from '../types/tokens';
+
+/**
+ * Helper method to convert token type to a SignableToken type
+ * @param token - the token type to convert to a SignableToken type
+ * @returns the converted SignableToken
+ */
+
```

Comment:
Does this annotation still work with this new line gap? Can we remove this?

Reply:
These methods are not exposed so you wouldn't find it in typedoc but yes it does work (see intellisense) with the line gap. I'll remove the gaps though.

Reply:
[05dab29](https://github.com/immutable/imx-core-sdk/pull/185/commits/05dab29a61dda6416ec0080c8c45a19130e36a68)